src/state/swap/hooks.tsx

```diff
@@ -133,7 +134,10 @@ export function useDerivedSwapInfo(): {
     [inputCurrency, outputCurrency]
   )
 
-  const allowedSlippage = useSwapSlippageTolerance(trade.trade ?? undefined)
+  // allowed slippage is either auto slippage, or custom user defined slippage if auto slippage disabled
```

Comment:
Neither of these function names refer to auto slippage. Is "auto slippage" === "slippage tolerance from trade"? Can one or the other be re-named?

Reply:
correct about "auto slippage" === "slippage tolerance from trade"

I like calling it autoSlippageTolerance - will rename that hook to match too